File: src/Services/API/Domain/StoredProcedure/ParameterDescriptionRequest.cs
namespace API.Domain.StoredProcedure
{
    public class ParameterDescriptionRequest
    {
        public string SchemaName { get; set; }
        public string StoredProcedureName { get; set; }
        public string ParameterName { get; set; }
        public string Description { get; set; }
    }

}

File: src/WebClient/mssql.client/src/app/procedure/model/ParameterDescriptionRequest.ts
// Represents a request to describe a parameter.
export class ParameterDescriptionRequest {
    schemaName: string;
    storedProcedureName: string;
    parameterName: string;
    description: string;
    constructor(init?: Partial<ParameterDescriptionRequest>) {
        Object.assign(this, init);
    }
}
